main.js

```diff
@@ -0,0 +1,12 @@
+function Book(title, author, pages, read){
+    this.title = "title";
+    this.author = "author";
+    this.pages = "pages";
+    this.read = "isRead";
```

Comment:
Kindly used Boolean value for.
```this.read = true```. Then when status is checked with ``` this.read ? "read" : "Not Read yet"```

Reply:
Thank you so much!
So you want me to use Boolean there, right?